common/djangoapps/philu_commands/management/commands/auto_generate_certificates_for_open_courses.py

```diff
@@ -3,11 +3,15 @@
 enrolled in currently running courses with early_no_info or early_with_info set
 in the certificate_display_behavior setting in course advanced settings
 """
-import json
+
 from logging import getLogger
 
 from django.core.management.base import BaseCommand
 
+from datetime import datetime
+from django.apps import apps
+from opaque_keys.edx.keys import UsageKey
+
```

Comment:
group imports properly
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
@zeshanarifios where should i move it :P

Reply:
Imports are grouped.